app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    text = command.split()
+    cp, from_file, to_file = text
+    if "cp" not in cp or len(text) != 3:
```

Comment:
```suggestion
    if "cp" != cp or len(text) != 3:
```

Reply:
`not in` operator uses to determine if a given value isn’t part of a collection of values. In your case, you must compare these two values.